index.html

```diff
@@ -1734,39 +1783,110 @@ <h3>Document Encoding</h3>
-            In the following example, a fictitious namespace <code>vendorm</code> from an "example vendor" is used
-            to provide document-level information not defined by DAPT.</p>
-          <pre class="example"
-            data-include="examples/metadata-proprietary.xml"
-            data-include-format="text">
-          </pre>
+            <a>Foreign vocabulary</a> MAY also be included,
+            either as attributes of <code>&lt;metadata&gt;</code> elements or
+            as descendant elements of <code>&lt;metadata&gt;</code> elements.</p>
+          <div class="note">
+            <p>It is possible to add information such as the title of the programme using [[TTML2]] constructs.</p>
+            <pre class="example"
+              data-include="examples/metadata-TTML2.xml"
+              data-include-format="text">
+            </pre>
+          </div>
+          <div class="note">
+            <p>It is possible to add workflow-specific information using a foreign namespace.
+              In the following example, a fictitious namespace <code>vendorm</code> from an "example vendor" is used
+              to provide document-level information not defined by DAPT.</p>
+            <pre class="example"
+              data-include="examples/metadata-proprietary.xml"
+              data-include-format="text">
+            </pre>
+          </div>
+          
+          <div class="advisement">
+            <strong>It is strongly recommended not to place data
+            whose semantics depend on the contents of the document
+            within <code>&lt;metadata&gt;</code> elements.</strong>
+            <p>Such data can be invalidated by <a>transformation processors</a> that
+            modify the contents of the document but preserve metadata
+            while being unaware of their semantics.
+            </p>
+          </div>
+
+          <section class="informative">
+            <h5>Defining and using foreign vocabulary that is not metadata</h5>
+            <p>If <a>foreign vocabulary</a> is included in locations other than <code>&lt;metadata&gt;</code> elements
+              it will be pruned by <a>transformation processors</a> that do not support features associated with that vocabulary,
+              as required in <a href="#unrecognised-elements-and-attributes"></a>.</p>
+            <p>A mechanism is provided to prevent such pruning, and to define semantics for such <a>foreign vocabulary</a>,
+              allowing it to be located outside a <code>&lt;metadata&gt;</code> element without being pruned,
+              and to indicate content and processor conformance:</p>
+            <ol>
+              <li>Define a profile including a feature definition for that semantic and vocabulary,
+            with a profile designator.</li>
+              <li>Signal document conformance to that profile using the <a href="#ttp-contentprofiles"></a></li>
+            </ol>
```

Comment:
That's the flip side though, rather than describing document conformance, the one in processorProfiles is about the document declaring that the processor needs to support some profile to process the document. I think it's fine to have both.